visidata/loaders/_pandas.py

```diff
@@ -19,25 +21,43 @@ def __getattr__(self, k):
 
 # source=DataFrame
 class PandasSheet(Sheet):
+    def dtype_to_type(self, dtype):
+        import numpy as np
+        try:
+            if np.issubdtype(dtype, np.integer):
+                return int 
+            if np.issubdtype(dtype, np.floating):
+              return float
+            if np.issubdtype(dtype, np.datetime64):
```

Comment:
does this work with datetime columns?  I'm not sure if I commented it out because it was the wrong mechanism (thanks for showing me the right idiom, btw), or because visidata.date and np.datetime64 aren't compatible.

Reply:
It does work with datetime columns, I work quite a bit with time series and have used them in visidata before :)